tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -242,6 +242,8 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
     pm.addNestedPass<FuncOp>(disc_ral::createDiscGpuConvPaddingLegalization());
   }
 
+  pm.addNestedPass<FuncOp>(disc_ral::createDiscDotMergePass());
```

Comment:
Does this pass use the placement information? if not, maybe we can put it right before the placer pass?

Reply:
It could be placed just after dot rewriter. By the way, are there side effects placer pass?

Reply:
The placer pass will attach an attribute to each mhlo ops. if other passes after the placer pass add new ops, then these ops will miss placement attributes.

Reply:
Done.